database/neo4j/neo4j.go

```diff
@@ -73,24 +73,14 @@ func (n *Neo4j) Open(url string) (database.Driver, error) {
 	uri.Scheme = "bolt"
 	msQuery := uri.Query().Get("x-multi-statement")
 
-	// Whether to turn on/off TLS encryption.
-	tlsEncrypted := uri.Query().Get("x-tls-encrypted")
```

Comment:
We should maintain support for encryption: https://github.com/neo4j/neo4j-go-driver/blob/4.2/MIGRATIONGUIDE.md#new-uri-scheme

Reply:
Neo4J version 4 removed the encrypted field in config.
https://github.com/neo4j/neo4j-go-driver/blob/4.2/neo4j/config.go#L33

This value was used to set in Neo4J config, and the encrypted field was deleted in the config at Neo4J version 4.
https://github.com/golang-migrate/migrate/pull/528/files/87dc261e9217cd1f0f11bde0b630b58e4c806a11#diff-fd7b1dbd62d757f42a6a55c25a7355d22f5290ef52267d24b23bf0864b51df08L104-L106

Reply:
You'll still need to parse this option and change the `uri.Scheme` accordingly. See this [table](https://neo4j.com/docs/go-manual/4.2/client-applications/#go-driver-configuration-examples). e.g. add `+s` or `+ssc`
Maybe if `x-tls-encrypted=self-signed`, we append `+ssc`

Reply:
Updated https://github.com/golang-migrate/migrate/pull/528/commits/b3878eecddc19781b52511d43b48fba02a364c61